Tests/FunctionalTests/EdgePathOverwriteTests.swift

```diff
@@ -0,0 +1,156 @@
+//
+// Copyright 2022 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+@testable import AEPCore
+@testable import AEPEdge
+import AEPEdgeIdentity
+import AEPServices
+import Foundation
+import XCTest
+
+/// End-to-end testing for the AEPEdge public APIs
+class AEPEdgePathOverwriteTests: FunctionalTestBase {
```

Comment:
can you also add a test that it does not include the "request" portion in the request body?

Reply:
I see the request being sent as event payload.

Reply:
hmm.. that should not be the case, we should handle this and not include the client-side "request" portion in the request to Konductor